projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.spec.ts

```diff
@@ -1786,17 +1794,21 @@ describe('IgxSimpleCombo', () => {
                 fixture.detectChanges();
                 combo = fixture.componentInstance.reactiveCombo;
             });
-            it('should not select null, undefined and empty string in a reactive form with required', () => {
+            // TODO: this test makes no sense - reactive combo with values equal to null, undefined and empty string
+            it('should not select null, undefined and empty string in a reactive form with required', fakeAsync(() => {
```

Comment:
Excluding the fact the `value` getter is weird, the form control integration actually works (or should) correctly and should be fed single items. Also the test is trying to _not_ select some values so the required validator triggers, no?